src/Sylius/Bundle/UserBundle/Resources/views/Security/login.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'SyliusUserBundle::layout.html.twig' %}
+
+{% block content %}
+<div class="page-header">
+    <h1>{{ 'sylius.ui.login'|trans }}</h1>
+</div>
+
+{% if error %}
+    <div class="alert alert-danger">
+        {{ error.messageKey|trans(error.messageData, 'security') }}
+    </div>
+{% endif %}
+
+<form action="{{ path('sylius_user_security_check') }}" method="post" class="form-horizontal" >
+    <fieldset>
+        {{ form_errors(form) }}
+
+        {{ form_row(form._username) }}
+        {{ form_row(form._password) }}
+        {{ form_rest(form) }}
+        <div class="form-group">
+            <div class="col-sm-offset-2 col-sm-9">
+                <div class="checkbox">
+                    <label for="remember_me">
+                        {{ 'sylius.form.user.remember_me'|trans }} &nbsp; <input type="checkbox" id="remember_me" name="_remember_me" value="on" />
+                    </label>
+                </div>
+            </div>
+        </div>
+        <div class="pull-right">
+            <p>
+                <a href="{{ path('sylius_user_request_password_reset_token') }}" class="btn btn-info btn-sm">{{ 'sylius.ui.forgot_password'|trans }}</a>
+                <a href="{{ path('sylius_user_registration') }}" class="btn btn-primary btn-sm">{{ 'sylius.ui.no_account'|trans }}</a>
+            </p>
+        </div>
+    </fieldset>
+
+    <div class="form-actions clearfix">
+        <button class="btn btn-primary btn-lg" type="submit">
+            <i class="icon-lock"></i>
+            {{ 'sylius.ui.login'|trans }}
+        </button>
+    </div>
+</form>
+{% endblock %}
```

Comment:
Same comment as above, I think this is too complex for default template. If we want to style it to match Sylius - let's do it in WebBundle. This has another disadvantage - people will look for these templates in WebBundle and find nothing, cause it is in User.